Dockerfile

```diff
@@ -1,5 +1,4 @@
-FROM debian:stretch
-
+FROM debian:buster
```

Comment:
Is this a good idea? I would argue that the package should be built from the oldest supported debian, not the newest. New releases are typically backwards compatible, but old releases are not forwards compatible.

Reply:
In this case we are using a custom toolchain and installing golang from upstream, so it is only liblzma-lib that we are using from Debian.

But anyway, in general terms I agree and we should get it as a general rule: to follow one stable Debian release behind the latest one.

I am dropping this commit.